app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+        self, comfort_class: int, clean_mark: int, brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = 'BMW'
```

Comment:
The brand is hardcoded to 'BMW', which means regardless of the brand passed to the constructor, it will always be set to 'BMW'. If you want to set the brand to the one provided in the constructor, you should use 'self.brand = brand' instead.